packages/resource-deployment/scripts/setup-all-pools-for-batch.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set -eo pipefail
+
+
+export resourceGroupName
+export batchAccountName
+export pool
+export keyVault
+export resourceName
+export systemAssignedIdentities
+export principalId
+export enableSoftDeleteOnKeyVault
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -r <resource group>
+"
+    exit 1
+}
+
+function waitForProcesses() {
+    local processesToWaitFor=$1
+
+    list="$processesToWaitFor[@]"
+    for pid in "${!list}"; do
+        echo "Waiting for process with pid $pid"
+        wait $pid
+        echo "Process with pid $pid exited"
+    done
+}
+
+function setupPools() {
+    # Enable managed identity on Batch pools
+    pools=$(az batch pool list --query "[].id" -o tsv)
+
+    echo "Setup system identity for created pools"
+    for pool in $pools; do
+        command=". ${0%/*}/enable-system-identity-for-batch-vmss.sh"
+        commandName="System identity for pool $pool"
```

Comment:
not correlate with command - do we need the command name?

Reply:
as mentioned above, it can have sensitve data.

Reply:
Still you need fix command name `Enable ...`